tests/templates.test.js

```diff
@@ -0,0 +1,118 @@
+import { createMocks } from "node-mocks-http";
+import templates from "../pages/api/templates";
+import client from "next-auth/client";
+import validFormTemplate from "./validFormTemplate.json";
+import brokenFormTemplate from "./brokenFormTemplate.json";
+
+import fetchMock from "jest-fetch-mock";
+
+global.TextEncoder = require("util").TextEncoder;
+global.TextDecoder = require("util").TextDecoder;
+
+jest.mock("next-auth/client");
+
+jest.mock("@aws-sdk/client-lambda", () => {
+  return {
+    LambdaClient: jest.fn(() => {
+      return {
+        send: jest.fn((command) => {
+          const encoder = new TextEncoder(),
+            decoder = new TextDecoder();
```

Comment:
If these are going to be on separate lines anyways. Might as well declare them in separate lines. I was a little confused reading this